### PR TITLE
feat(git): add git unpushed alias

### PR DESCRIPTION
**Pull request by georgettica**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ repos:
       - id: mixed-line-ending
       - id: check-added-large-files
   - repo: https://github.com/jumanjihouse/pre-commit-hooks
-    rev: 2.1.4
+    rev: 2.1.5
```
Comment (davidpfarrell) [inline .pre-commit-config.yaml:18]: This change doesn't belong in this PR - Maybe submit it as a separate PR for consideration.

File: aliases/available/git.aliases.bash [hunk 2]
```diff
@@ -118,7 +119,7 @@ alias gswt="git switch --track"
 # Git home
 alias ghm='cd "$(git rev-parse --show-toplevel)"'
 if ! _command_exists gh; then
-  alias gh='ghm'
+	alias gh='ghm'
```
Comment (davidpfarrell) [inline aliases/available/git.aliases.bash:122]: These whitespace changes seem unrelated to the PR

**Comment by georgettica** [comment]
i'll do as you say, as I guess I didn't rebase on master :)
thanks for the notes!

File: .pre-commit-config.yaml (hunk 1 above)
Comment (georgettica) [reply .pre-commit-config.yaml:18]: will do

File: aliases/available/git.aliases.bash [hunk 3]
```diff
@@ -18,6 +18,7 @@ alias gbd='git branch -d'
 alias gbm='git branch -m'
 alias gbt='git branch --track'
 alias gdel='git branch -D'
+alias gbup='git log --branches --not --remotes --no-walk --decorate --oneline' # FROM https://stackoverflow.com/questions/39220870/in-git-list-names-of-branches-with-unpushed-commits
```
Comment (davidpfarrell) [inline aliases/available/git.aliases.bash:21]: I believe we keep these aliases alphabetized within each sub-group.
So you can move this up between `gbt` and and `gdel`.
Other than that, looks great !

Comment (georgettica) [reply aliases/available/git.aliases.bash:21]: Sure will do that in the UI now!

**Review by davidpfarrell — APPROVED** [review]
Great thanks - Approved !

**Review by NoahGorny — CHANGES_REQUESTED** [review]
Hi @georgettica, nice addition!
I would prefer if you put this new alias in the `git log` aliases sections, which starts at line 70.
Do that and its good to go :smile:

**Comment by georgettica** [comment]
will move then

**Review by NoahGorny — APPROVED** [review]
well done @georgettica :smile:
